scala-mode.el

```diff
@@ -30,7 +30,7 @@
            emacs-major-version  emacs-minor-version)))
 
 (defgroup scala nil
-  "A programming mode for the Scala language 2.9"
+  "A programming mode for the Scala language 2/3"
```

Comment:
Scala language two thirds? 😄

Reply:
Lol yeah that does look funny. What about `2 and 3`?

Reply:
I like it!